File: Orenes/Models/PedidoEntregado.cs
namespace Orenes.Models
{
    public class PedidoEntregado
    {
        public int PedidoEntregadoId { get; set; }
        public int ClienteId { get; set; }
        public string DireccionEntrega { get; set; }
        public EstadoPedido Status { get; set; }
        public Cliente Cliente { get; set; }
    }
}

File: PRUEBATECNICA/angular-Pruebatecnica/src/app/API/models/pedido-entregado.ts
/* tslint:disable */
/* eslint-disable */
import { Cliente } from './cliente';
import { EstadoPedido } from './estado-pedido';
export interface PedidoEntregado {
  cliente?: Cliente;
  clienteId?: number;
  direccionEntrega?: null | string;
  pedidoEntregadoId?: number;
  status?: EstadoPedido;
}
